### PR TITLE
Added response for expense submission

### PR DESCRIPTION
**Pull request by bzimmer**
Return the submitted expenses with their transaction ids.

### CODE REVIEW

File: expense_test.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestExpeseService_Create(t *testing.T) {
+func TestExpenseService_Create(t *testing.T) {
```
Comment (lukasmalkmus) [inline expense_test.go:15]: That seems like a typo? The original name was fine.

**Review by lukasmalkmus — CHANGES_REQUESTED** [review]
Thanks for the PR!

There is one typo I guess, please review that.

**Comment by lukasmalkmus** [comment]
My bad,  I read that wrong :)

Thanks for the PR 👍

**Comment by bzimmer** [comment]
Thanks for starting the project!

**Pull request merged**